### ISSUE TITLE
Nginx not adding host after Homestead reload

### ISSUE THREAD

**Issue by magarrent** [hyn/multi-tenant#644]
__Description__

Hey there, when I stop and run up Homestead, the host that I had with tenant disappear, and I have to delete and recreate BDs all time

__Information__

- hyn/multi-tenant version: 5.2.7
- laravel version: 5.6
- database driver and version: 5.7.22-0ubuntu18.04.1
- webserver software and version: nginx 1.14
- php version: 7.2

---

__tenancy.php config__

DEFAULT

__webserver.php config__

DEFAULT

---

**Comment by luceos** [comment]
I'm no expert on Homestead/Vagrant. I'm not sure what happens if you modify an nginx configuration and re-run the instance. It might as well use the configuration instead of the created vhost configuration, because those files are only contained inside the container..

Anyone?

**Comment by luceos** [comment]
PS you set "webserver.php config" to DEFAULT, but the default is having NGINX not enabled. If you don't take the time to be thorough, why should we?

**Comment by magarrent** [comment]
> PS you set "webserver.php config" to DEFAULT, but the default is having NGINX not enabled. If you don't take the time to be thorough, why should we?

Hey, thanks for the response. Sorry, in my webserver.php I have:

apache => false
nginx => true

**Comment by LDP-Dries** [comment]
Hey,

Which commands are you using to power off and up the Homestead environment? Are you using the CMD as Administrator or normal user?

**Comment by magarrent** [comment]
> Hey,
> 
> Which commands are you using to power off and up the Homestead environment? Are you using the CMD as Administrator or normal user?


From a normal user in my Real machine:

```
vagrant up
vagrant halt
```

That's all

**Comment by luceos** [comment]
TLDR if the vhosts exists before running halt and disappear when you run up again, it's definitely not an issue with this package but with vagrant. I can't provide support in our issue tracker for other tools.

Feel free to join discord and ask the question there, or get assistance where knowledge about vagrant is more readily available.

**Comment by magarrent** [comment]
Hey finally I got the solution without recreate DBs, when vagrant is UP, enter vagrant ssh and reload nginx:

`sudo service nginx reload`
